src/Microsoft.AspNetCore.ResponseCompression/BodyWrapperStream.cs

```diff
@@ -133,67 +138,101 @@ public override void Write(byte[] buffer, int offset, int count)
         }
 
 #if NET451
-        public override IAsyncResult BeginWrite(byte[] buffer, int offset, int count, AsyncCallback callback, object state)
+        public override IAsyncResult BeginWrite(byte[] buffer, int offset, int size, AsyncCallback callback, object state)
```

Comment:
size? it's count in the base class.
https://msdn.microsoft.com/en-us/library/system.io.stream.beginwrite(v=vs.110).aspx